tests/test_syntax/blocks/test_tables.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
```

Comment:
As this is for an extension, let's put it at `tests/test_syntax/extensions/test_tables.py`.

Reply:
Will do.